examples/22_define-router/src/entries.tsx

```diff
@@ -193,4 +193,17 @@ export default defineRouter({
     }
     throw new Error('renderRoute: No such path:' + path);
   },
+  getApiConfig: async () => [
+    {
+      path: [
+        { type: 'literal', name: 'api' },
+        { type: 'literal', name: 'hi.txt' },
```

Comment:
The extension is necessary, if `isStatic=true`. Otherwise, unnecessary, I think.